### PR TITLE
Add chapter about TYPO3 Version and which to start

### PR DESCRIPTION
**Pull request by mkiebele**

### CODE REVIEW

File: Documentation/Installation/Version.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+:navigation-title: TYPO3 Version
+.. include:: /Includes.rst.txt
+.. _typo3-version:
+
+================================
+TYPO3 Version and which to start
```
Comment (linawolf) [inline Documentation/Installation/Version.rst:6]:
```suggestion
TYPO3 version and which to start
```

**Review by linawolf — APPROVED** [review]

File: Documentation/Installation/Version.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
+:navigation-title: TYPO3 Version
+.. include:: /Includes.rst.txt
+.. _typo3-version:
```
Comment (froemken) [inline Documentation/Installation/Version.rst:3]:
```suggestion
..  include:: /Includes.rst.txt
..  _typo3-version:
```

File: Documentation/Installation/Version.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+================================
+TYPO3 version and which to start
+================================
+
+To get started with TYPO3, it’s recommended to use the latest Long-Term Support
```
Comment (froemken) [inline Documentation/Installation/Version.rst:9]: IMO we don't use the short form `it's` or `don't` in TYPO3 docs.

```suggestion
To get started with TYPO3, it is recommended to use the latest Long-Term Support
```

Comment (froemken) [reply Documentation/Installation/Version.rst:9]: On typo3.com the wording of LTS is:
```suggestion
To get started with TYPO3, it’s recommended to use the latest Long Term Support
```

**Pull request merged**